pysat/instruments/methods/testing.py

```diff
@@ -102,72 +103,113 @@ def initialize_test_meta(epoch_name, data_keys):
     meta['longitude'] = {'units': 'degrees', 'long_name': 'Longitude',
-                         'value_min': 0.0, 'value_max': 360.0}
+                         'value_min': 0.0, 'value_max': 360.0,
+                         'desc': 'Geographic Longitude'}
     meta['latitude'] = {'units': 'degrees', 'long_name': 'Latitude',
-                        'value_min': -90.0, 'value_max': 90.0}
-    meta['altitude'] = {'units': 'km', 'long_name': 'Altitude'}
+                        'value_min': -90.0, 'value_max': 90.0,
+                        'desc': 'Geographic Latituce'}
+    meta['altitude'] = {'units': 'km', 'long_name': 'Altitude',
+                        'value_min': 0.0, 'value_max': np.inf,
+                        'desc': 'Height above mean Earth.'}
     meta['orbit_num'] = {'units': '', 'long_name': 'Orbit Number',
-                         'desc': 'Orbit Number', 'value_min': 0.0,
-                         'value_max': 25000.0,
+                         'desc': 'Orbit Number', 'value_min': 0,
+                         'value_max': 25000, 'fill': -1,
                          'notes': ''.join(['Number of orbits since the start ',
                                            'of the mission. For this ',
                                            'simulation we use the number of ',
                                            '5820 second periods since the ',
                                            'start, 2008-01-01.'])}
 
+    meta['dummy1'] = {'value_min': 0, 'value_max': 24, 'fill': -1}
+    meta['dummy2'] = {'value_min': 0, 'value_max': 24, 'fill': -1}
+    meta['dummy3'] = {'value_min': 0., 'value_max': 24024.}
+    meta['dummy4'] = {'desc': 'Dummy variable - UTS like', 'value_min': 0.,
+                      'value_max': 86400., 'fill': np.nan}
+
+    meta['unicode_dummy'] = {'desc': 'Dummy unicode variable.', 'units': ''}
+    meta['string_dummy'] = {'desc': 'Dummy string variable.', 'units': ''}
+
+    meta['dummy_drifts'] = {'desc': 'Dummy drift values.', 'value_min': -1000.,
+                            'value_max': 1000., 'fill': np.nan}
+    # Add metadata for integer dummy variables.
+    meta_dict = {'value_min': 0, 'value_max': 2, 'fill': -1}
+    var_list = ['int8_dummy', 'int16_dummy', 'int32_dummy', 'int64_dummy']
+    for var in var_list:
+        meta[var] = meta_dict
+
     # Standard metadata required for xarray.
-    meta['profiles'] = {'long_name': 'profiles'}
+    meta['profiles'] = {'long_name': 'profiles', 'value_min': 0,
+                        'value_max': 4294967295, 'fill': -1,
+                        'desc': ''.join(['Testing profile multi-dimensional ',
+                                         'data indexed by time.']),
+                        'notes': ''.join(['Note the value_max is largest ',
+                                          'netCDF4 supports, but is ',
+                                          'lower than actual 64-bit int limit.',
+                                          ])}
```

Comment:
Stylistically, I don't like this extra comma and hanging closing parens.  Recommend breaking the text up a bit more or breaking the lasts join list on the opening square bracket.